doc/background/index.md

```diff
@@ -0,0 +1,50 @@
+# Background
+
+The Background section aims to clarify, deepen, and broaden the understanding of Panel with discussions about topics at a high-level or from alternate angles. This may include reasoning about design decisions, historical development, and technical constraints.
+
+Beyond the [Getting Started > Core Concepts](../getting_started/core_concepts.md) that new users must internalize in order to begin working, this Background section is intended to help practitioners form and strengthen a conceptual web that facilitates new and advanced usage directions.
+
+::::{grid} 1 2 2 3
+:gutter: 1 1 1 2
+
+:::{grid-item-card} {octicon}`rows;2.5em;sd-mr-1` Components
+:link: components/components_overview
+:link-type: doc
+
+Deepen your understanding about Panel's visible objects and layouts types.
+:::
+
+:::{grid-item-card} {octicon}`workflow;2.5em;sd-mr-1` APIs
+:link: api/api
+:link-type: doc
+
+Explore the reasoning for each of Panel's APIs.
+:::
+
+:::{grid-item-card} {octicon}`multi-select;2.5em;sd-mr-1` Param
+:link: param/param
+:link-type: doc
+
+Discover why Panel utilizes the Param library.
+:::
+
+:::{grid-item-card} {octicon}`git-compare;2.5em;sd-mr-1` Library Comparisons
+:link: ../about/comparisons
```

Comment:
Should we move that section from about -> background?

Reply:
I'm not fully convinced we need an About page tbh.

Reply:
ok, we can save the extermination of About for another day, but I've moved comparisons to background